### PR TITLE
feat(api): add List Strings on Workflow Step method (#184)

### LINKED ISSUE #184: Workflows API: new `List Strings on the Workflow Step` method
The Workflows API (Crowdin Enterprise) has a new method to list strings on the workflow step. It should also be added to the Crowdin API clients.

**References:**

- [List Strings on the Workflow Step](https://support.crowdin.com/developer/enterprise/api/v2/#tag/Workflows/operation/api.projects.workflow-steps.strings.getMany)

### PR DESCRIPTION
**Pull request by imprvhub**
This commit implements API endpoint to list strings on workflow steps with full test coverage:

- Add list_workflow_step_strings method with query parameters support
- Implement path helper get_workflow_step_strings_path
- Add parameterized tests covering basic usage, full parameters and edge cases
- Add mock tests for API request validation
- Ensure all test cases follow existing patterns in the codebase
- Follow Crowdin Enterprise API specification

Closes #184

### CODE REVIEW

File: crowdin_api/api_resources/workflows/resource.py [hunk 1]
```diff
@@ -37,7 +37,6 @@ def list_workflow_steps(self, projectId: Optional[int] = None):
         Link to documentation:
         https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.workflow-steps.getMany
         """
-
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/workflows/resource.py:40]: Please avoid these unnecessary changes

**Comment by andrii-bodnar** [comment]
@imprvhub hmm, it's interesting why it didn't fail before. Probably the rules have changed over time and it only fails on updated files or something. Ok, let's make it follow the validation rules.

**Comment by imprvhub** [comment]
@andrii-bodnar Done!.
